sdk/python/feast/repo_config.py

```diff
@@ -82,6 +82,10 @@ class Config:
 class RegistryConfig(FeastBaseModel):
     """Metadata Store Configuration. Configuration that relates to reading from and writing to the Feast registry."""
 
+    registry_type: StrictStr = "file"
```

Comment:
why do we have this if registry store type does the same thing

Reply:
Store type doesn't do the same thing cause it abstracts over where the registry proto is saved. It's API is basically `save_registry_proto` and `get_registry_proto` which is not as fine grained as the individual Registry operations.